src/plugins/links/public/components/editor/links_panel_editor.tsx

```diff
@@ -29,36 +29,36 @@ import {
 } from '@elastic/eui';
 import { DashboardContainer } from '@kbn/dashboard-plugin/public/dashboard_container';
 
-import { NavigationLayoutInfo } from '../../embeddable/types';
+import { LinksLayoutInfo } from '../../embeddable/types';
 import {
-  NavigationEmbeddableLink,
-  NavigationLayoutType,
-  NAV_HORIZONTAL_LAYOUT,
-  NAV_VERTICAL_LAYOUT,
+  LinksLink,
+  LinksLayoutType,
+  LINKS_HORIZONTAL_LAYOUT,
+  LINKS_VERTICAL_LAYOUT,
 } from '../../../common/content_management';
 import { coreServices } from '../../services/kibana_services';
-import { NavEmbeddableStrings } from '../navigation_embeddable_strings';
+import { LinksStrings } from '../links_strings';
 import { openLinkEditorFlyout } from '../../editor/open_link_editor_flyout';
-import { memoizedGetOrderedLinkList } from '../../editor/navigation_embeddable_editor_tools';
-import { NavigationEmbeddablePanelEditorLink } from './navigation_embeddable_panel_editor_link';
-import { NavigationEmbeddablePanelEditorEmptyPrompt } from './navigation_embeddable_panel_editor_empty_prompt';
+import { memoizedGetOrderedLinkList } from '../../editor/links_editor_tools';
+import { LinksPanelEditorLink } from './links_panel_editor_link';
+import { LinksPanelEditorEmptyPrompt } from './links_panel_editor_empty_prompt';
 
 import { TooltipWrapper } from '../tooltip_wrapper';
 
-import './navigation_embeddable_editor.scss';
+import './links_editor.scss';
 
 const layoutOptions: EuiButtonGroupOptionProps[] = [
   {
-    id: NAV_VERTICAL_LAYOUT,
-    label: NavigationLayoutInfo[NAV_VERTICAL_LAYOUT].displayName,
+    id: LINKS_VERTICAL_LAYOUT,
+    label: LinksLayoutInfo[LINKS_VERTICAL_LAYOUT].displayName,
   },
   {
-    id: NAV_HORIZONTAL_LAYOUT,
-    label: NavigationLayoutInfo[NAV_HORIZONTAL_LAYOUT].displayName,
+    id: LINKS_HORIZONTAL_LAYOUT,
+    label: LinksLayoutInfo[LINKS_HORIZONTAL_LAYOUT].displayName,
   },
 ];
 
-const NavigationEmbeddablePanelEditor = ({
+const LinksPanelEditor = ({
```

Comment:
```suggestion
const LinksEditor = ({
```

+ rename file to `links_editor.tsx`